_maps/configs/raven.json

```
@@ -0,0 +1,21 @@
+{
+	"map_name": "Raven-class Heavy Transporter",
+	"map_short_name": "Raven-class",
+	"map_path": "_maps/shuttles/shiptest/nanotrasen_raven.dmm",
+	"map_id": "nanotrasen_raven",
+	"job_slots": {
+		"Captain": 1,
+		"Head of Security": 1,
+		"Chief Medical Officer": 1,
+		"Head Of Personnel": 1,
```

```suggestion
		"Head of Personnel": 1,
```
can't believe I made this case-sensitive